dll/cpl/input/settings_page.c

```diff
@@ -453,8 +598,7 @@ SettingsPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
             break;
 
         case WM_NOTIFY:
-            OnNotifySettingsPage(hwndDlg, lParam);
-            break;
+            return OnNotifySettingsPage(hwndDlg, lParam);
```

Comment:
Does this function need to return values that are potentially different from zero?

Reply:
In order to prevent Collapse in future.

Reply:
NOTE: `comctl32!TREEVIEW_Collapse` function has a bug. That function didn't check the return value of `TREEVIEW_SendExpanding`.

Reply:
https://bugs.winehq.org/show_bug.cgi?id=53727